### ISSUE TITLE
PDF export available?

### ISSUE THREAD

**Issue by TribuneX** [lakoliu/Furtherance#126]
Does the iOS App support to export times on a monthly basis as a PDF file?

**Comment by rickykresslein** [comment]
No, at the moment it only exports to CSV. PDF export is on the list of future features, but unfortunately I can't give a time estimate except to say there are several features ahead of it on the list. But hopefully it wont be long 😊

**Comment by TribuneX** [comment]
Thanks! Is the roadmap / feature list somewhere accessible?

**Comment by rickykresslein** [comment]
No, there isn't. Perhaps I will post one in the future, but right now it's just a kanban that lives on my home server. I change it up constantly, so it wouldn't be very helpful to post in its current state, I think.
